### DISCUSSION TITLE
New to GSOC contribution, wanted to know more about the Tech

### DISCUSSION THREAD

**Discussion by shashankyadahalli** [uaanchorage/GSoC#5] [General]
I am Shashank, I am new to GSOC contributions, but I am well versed with matlab, python,Java just wanted to know, what to know what all the things the organization looking for

**Comment by pradeeban** [comment]
Hi,
Please review this repository, the provided links to resources, previous discussions, and individual repositories of the project ideas you are interested in. That gives details specific to our organization.

Then, also review the official GSoC contributor guides and FAQs released by Google. That will inform you of general GSoC expectations.

Those should give you plenty of information on GSoC.
